### ISSUE TITLE
[feature]: more than one github repo integration

### ISSUE THREAD

**Issue by BayCo** [makeplane/plane#1255]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

I would like to integrate more than one github repo integration to one project

### Why should this be worked on?

One project could have more than one related github repo's (frontend, backend, microservices etc.)

**Comment by rhea0110** [comment]
Hey @BayCo, we sincerely appreciate and value your suggestion. Thank you for submitting the feature request. Our team will brainstorm around it, and we will keep you updated on any progress made.

Thanks!

**Comment by gthay** [comment]
Yeah would be awesome to be able to connect multiple GitHub Orgs to one Plane Workspace. So freelancer can aggregate their different project into one workspace and manage it from there.
